.github/workflows/polyglot_build.yml

```diff
@@ -8,37 +8,48 @@ env:
   CARGO_TERM_COLOR: always
 jobs:
   build_and_test:
-    name: Piranha
+    name: Polyglot Piranha Build
```

Comment:
Can we run all the steps when `if: ${{ steps.filter.outputs.polyglot == 'true' }}` instead of adding this condition to each step?